hoomd/md/minimize/fire.py

```diff
@@ -168,25 +171,33 @@ def __init__(self,
                     "min_steps_conv": 10
                 }))
 
+    def _validate_methods(self):
+        """Ensure that only NVE/NPH integration methods are used."""
+        for method in self.methods:
+            method_name = method.__class__.__name__
+            if method_name not in ["NVE", "NPH"]:
+                raise RuntimeError("{} integration method not compatible with "
+                                   "FIRE energy minimizer".format(method_name))
+
     def _attach(self):
+        self._validate_methods()
+
         if isinstance(self._simulation.device, hoomd.device.CPU):
             cls = getattr(_md, self._cpp_class_name)
         else:
             cls = getattr(_md, self._cpp_class_name + "GPU")
         self._cpp_obj = cls(self._simulation.state._cpp_sys_def, self.dt)
         super()._attach()
 
-    def get_energy(self):
-        """Returns the energy after the last iteration of the minimizer."""
-        return self._cpp_obj.getEnergy()
-
-    def has_converged(self):
-        """Test if the energy minimizer has converged.
+    @log(requires_run=True)
+    def energy(self):
+        """float: Get the energy after the last iteration of the minimizer."""
+        return self._cpp_obj.energy
 
-        Returns:
-            True when the minimizer has converged. Otherwise, return False.
-        """
-        return self._cpp_obj.hasConverged()
+    @log(default=False, category='object')
```

Comment:
```suggestion
    @log(default=False)
```
`bool` will errorlessly convert to a scalar 1 for `True` 0 for `False`. This will allow loggers that only support scalars to handle Boolean values as well, by treating them as numbers by default (of course any backend is free to handle them as desired).